src/inc-upgrade.php

```diff
@@ -833,6 +833,14 @@ function lovd_addConditionalSQL ($sCondition, $aConditionArgs, $sSQL)
                       VALUES ("' . $_CONF['refseq_build'] . '", "' . $_CONF['refseq_build'] . ' / ' .
                               $_SETT['human_builds'][$_CONF['refseq_build']]['ncbi_name'] . '", 0, NOW())',
                  ),
+                 '3.5-pre-03' => array(
+                     'ALTER TABLE ' . TABLE_VARIANTS .
+                     'ADD COLUMN source VARCHAR(4) ',
```

Comment:
Hier mist nog een locatie waar de kolom moet worden toegevoegd; `AFTER type`.
Ook past deze query denk ik op één regel; dat heeft ook de voorkeur. Als het niet past, spring dan `ADD COLUMN` wat in zodat het zichtbaar is dat het bij de vorige regel hoort.